plugin/latexlivepreview.vim

```diff
@@ -92,6 +93,37 @@ function! s:StartPreview(...)
 
     let b:livepreview_buf_data['py_exe'] = s:py_exe
 
+    " This Pattern Matching will FAIL for multiline biblatex declarations,
+    " in which case the `g:livepreview_use_biber` setting must be respected.
+    let l:general_pattern = '^\\usepackage\[.*\]{biblatex}'
+    let l:specific_pattern = 'backend=bibtex'
+    let l:position = search(l:general_pattern, 'cw')
+    if ( l:position != 0 )
+        let l:matches = matchstr(getline(l:position), specific_pattern)
+        if ( l:matches == '' )
+            " expect s:useBiber=1
+            if ( s:useBiber == 0 )
```

Comment:
Let's use snake case here instead of camel case, in order to keep it consistent with other variables